server/handler.go

```diff
@@ -66,3 +72,44 @@ func (e *env) healthHandler(c *gin.Context) {
 		"buildTime": version.BuildTime,
 	})
 }
+
+func (e *env) loginHandler(c *gin.Context) {
+	email := c.PostForm("email")
+	password := c.PostForm("password")
+
+	token, err := e.auth.GenToken(email, password)
+	if err != nil {
+		if err == users.ErrNoSuchUser {
+			c.String(http.StatusUnauthorized, "wrong credentials")
+			return
+		}
+		c.Error(errors.Wrap(err, "could not generate token"))
```

Comment:
returned error not checked

Reply:
?!

Reply:
I think proble is that it will be more readable if we will do it like that:
```
token, err := e.auth.GenToken(email, password)
if err == users.ErrNoSuchUser {
    // todo
}
if err != nil {
     // todo
}
```

Reply:
> ?!
Please see the signature of function
https://godoc.org/github.com/gin-gonic/gin#Context.Error